assets/css/email-notifications/email-style.scss

```diff
@@ -35,6 +35,10 @@ $MOBILE_HEADER_SIZE: 32px;
 	float: right;
 }
 
+.wp-block-image img {
+	height: 100%;
```

Comment:
I'm not sure if it makes a difference but probably `height: auto` is more correct. WDYT?

Reply:
Thanks, Miro, I'll try it.

Reply:
Yes, it works. Updated here: https://github.com/Automattic/sensei/pull/7595/commits/6a5e1a746994ec97c295c2033d169bc62b8a6caa